streaming-connect-sink/src/test/java/com/adobe/platform/streaming/integration/AbstractConnectorTest.java

```diff
@@ -119,12 +124,13 @@ public void stop() {
     connect.stop();
   }
 
-  public void inletSuccessfulResponse() throws JsonProcessingException {
+  public void inletSuccessfulResponse() throws JsonProcessingException, HttpException {
     wiremockExtension.getWireMockServer()
       .stubFor(WireMock
       .post(WireMock.urlEqualTo(getRelativeUrl()))
       .willReturn(ResponseDefinitionBuilder.responseDefinition()
-      .withJsonBody(JacksonFactory.OBJECT_MAPPER.readTree("{\"payloadReceived\": true}"))));
+      .withJsonBody(JacksonFactory.OBJECT_MAPPER.readTree(HttpUtil.streamToString(this.getClass().getClassLoader()
```

Comment:
HttpUtil.streamToString is not required ObjectMapper can directly read from inputStream.
```suggestion
JacksonFactory.OBJECT_MAPPER.readTree(this.getClass()
  .getClassLoader()      
.getResourceAsStream(AEP_CONNECTOR_INLET_SUCCESSFUL_RESPONSE_CONFIG)      
```